source/install/config-ssl-http2-apache2.rst

```diff
@@ -0,0 +1,66 @@
+.. _config-ssl-http2-apache2:
+
+Configuring Apache2 with SSL and HTTP/2 (Unofficial)
+=====================================================
+
+.. important:: This unofficial guide is maintained by the Mattermost community and this deployment configuration is not yet officially supported by Mattermost, Inc. Community testing, feedback and improvements are welcome and greatly appreciated. You can `edit this page on GitHub <https://github.com/mattermost/docs/blob/master/source/install/config-ssl-http2-apache2.rst>`_.
+
+Once you've configured Apache2 as a proxy for your Mattermost Server, the easiest way to enable SSL on Apache2 is via Let's Encrypt and `Certbot <https://certbot.eff.org/#ubuntuxenial-apache>`_.
+
+Once installed, simply run:
+``$ certbot --apache`` and follow the straight forward guide. Afterwards you should find a new configuration file in ``/etc/apache2/sites-available`` which should follow the format: ``mysubdomain.mydomain.com-le-ssl.conf``.
+
+When opened, edit it to look something like the following:
```

Comment:
Can there be variations on how the file looks like? Or is this how it should be?